### PR TITLE
added import_csv

### PR DESCRIPTION
**Pull request by Aleisha02**

### CODE REVIEW

File: hail/python/hail/methods/impex.py [hunk 1]
```diff
@@ -2975,3 +2975,38 @@ def import_avro(paths, *, key=None, intervals=None):
...
+                comment=(),
+                missing="NA",
+                types={},
+                skip_blank_lines=False,
+                force_bgz=False,
+                filter=None,
+                find_replace=None,
+                force=False,
+                source_file_field=None) -> Table:
+   """Import csv file as :class:`.Table`."""
```
Comment (danking) [inline hail/python/hail/methods/impex.py:3008]: Let's add an example!

**Review by danking — CHANGES_REQUESTED** [review]

File: hail/python/hail/methods/impex.py [hunk 2]
```diff
@@ -2975,3 +2975,151 @@ def import_avro(paths, *, key=None, intervals=None):
         with DataFileReader(avro_file, DatumReader()) as data_file_reader:
             tr = ir.AvroTableReader(avro.schema.parse(data_file_reader.schema), paths, key, intervals)
     return Table(ir.TableRead(tr))
+
+
+@typecheck(paths=oneof(str, sequenceof(str)),
+          key=table_key_type,
```
Comment (danking) [inline hail/python/hail/methods/impex.py:2981]: Can you add one space before every line here? The `key` should line up with the `paths`.

**Review by danking — CHANGES_REQUESTED** [review]
This looks great, just one nit.

**Comment by Aleisha02** [comment]
ready for new review i think

File: hail/python/hail/methods/impex.py [hunk 3]
```diff
@@ -2975,3 +2975,151 @@ def import_avro(paths, *, key=None, intervals=None):
...
+           missing=oneof(str, sequenceof(str)),
+           types=dictof(str, hail_type),
+           skip_blank_lines=bool,
+           force_bgz=bool,
+           filter=nullable(str),
+           find_replace=nullable(sized_tupleof(str, str)),
+           force=bool,
+           source_file_field=nullable(str))
+def import_csv(paths,
+                key=None,
```
Comment (danking) [inline hail/python/hail/methods/impex.py:2995]: too many spaces now; the `k` in key should be directly under the `p` in paths.

File: hail/python/hail/methods/impex.py [hunk 4]
```diff
@@ -2975,3 +2975,151 @@ def import_avro(paths, *, key=None, intervals=None):
...
+                comment=(),
+                missing="NA",
+                types={},
+                skip_blank_lines=False,
+                force_bgz=False,
+                filter=None,
+                find_replace=None,
+                force=False,
+                source_file_field=None) -> Table:
+   """Import csv file as :class:`.Table`.
```
Comment (danking) [inline hail/python/hail/methods/impex.py:3008]: import "a" csv file as "a" ...

File: hail/python/hail/methods/impex.py [hunk 5]
```diff
@@ -2975,3 +2975,151 @@ def import_avro(paths, *, key=None, intervals=None):
...
+
+   Examples
```
Comment (danking) [inline hail/python/hail/methods/impex.py:3010]: need one space before the `E` in examples.

**Review by danking — CHANGES_REQUESTED** [review]

**Review by danking — APPROVED** [review]

**Pull request merged**